addons/project_todo/static/tests/delete_todo_stage_in_kanban_test.js

```diff
@@ -0,0 +1,79 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+import { addModelNamesToFetch } from "@bus/../tests/helpers/model_definitions_helpers";
+
+import { start } from "@mail/../tests/helpers/test_utils";
+
+import {
+    editInput,
+    click,
+    getFixture,
+} from "@web/../tests/helpers/utils";
+import { setupViewRegistries } from "@web/../tests/views/helpers";
+
+addModelNamesToFetch(["project.project", "project.task", "project.task.type"]);
+
+let target;
+
+QUnit.module('kanban todo stage deletion', {
+    beforeEach: async function () {
+        const pyEnv = await startServer();
+        const projectId = pyEnv["project.project"].create([
+            { name: "Project One" },
+        ]);
+        const userId = pyEnv["res.users"].create([
+            { name: "User One", login: 'one', password: 'one' },
+        ]);
+        const stageId = pyEnv["project.task.type"].create([
+            { name: "Personal stage 1", user_id: userId },
+        ]);
+        pyEnv["project.task"].create([
+            { name: 'task one', project_id: projectId, personal_stage_type_id: stageId, user_ids: [userId] },
+        ]);
+        target = getFixture();
+        setupViewRegistries();
+    }
+}, function () {
```

Comment:
We can also write it like this:

```suggestion
QUnit.module('Views', (hooks) => {
    hooks.beforeEach(async () => {
        const pyEnv = await startServer();
        const projectId = pyEnv["project.project"].create([
            { name: "Project One" },
        ]);
        const userId = pyEnv["res.users"].create([
            { name: "User One", login: 'one', password: 'one' },
        ]);
        const stageId = pyEnv["project.task.type"].create([
            { name: "Personal stage 1", user_id: userId },
        ]);
        pyEnv["project.task"].create([
            { name: 'task one', project_id: projectId, personal_stage_type_id: stageId, user_ids: [userId] },
        ]);
        target = getFixture();
        setupViewRegistries();
    });
    
    QUnit.module("ProjectTaskKanbanView");
}
```

(It is another way to write the test, maybe the new way 🤔 )🙂